src/components/ChatRoom.js

```diff
@@ -101,6 +120,7 @@ export default function ChatRoom() {
               <td>{chatroom.name}</td>
               <td>{chatroom.members.map((member) => users[member]).join(", ")}</td>
               <td>{users[chatroom.created_by]}</td>
+              <td onClick={() => handleDelete(chatroom.id)}>delete</td>
```

Comment:
Hi Harmeet, can you please make a delete button inside this column which should be a delete icon with red color and call handle delete on click on the icon.